### PR TITLE
missing books from spellbook.lua

### PR DESCRIPTION
**Pull request by l04d**

### CODE REVIEW

File: data/actions/actions.xml [hunk 1]
```diff
@@ -132,6 +132,11 @@
 	<action itemid="6120" script="other/spellbook.lua" />
 	<action fromid="8900" toid="8904" script="other/spellbook.lua" />
 	<action itemid="8918" script="other/spellbook.lua" />
+	<action fromid="22422" toid="22424" script="other/spellbook.lua" />
+	<action itemid="18401" script="other/spellbook.lua" />
+	<action itemid="16112" script="other/spellbook.lua" />
+	<action itemid="15411" script="other/spellbook.lua" />
```
Comment (marksamman) [inline data/actions/actions.xml]: Depth scutum should not work as a spellbook. Could you also re-order them in ascending order of the item IDs? Please squash your commits when you're done.

**Comment by l04d** [comment]
@marksamman Oh sorry for the mess, about the depth scutum, I'd never notice if you don't tell me, it's funny because I used to use it so much and never noticed =O

**Comment by marksamman** [comment]
It's OK, just squash your commits and write a commit message that better describes what you have changed. "Update filename" is vague.

File: data/actions/actions.xml [hunk 2]
```diff
@@ -132,7 +132,11 @@
 	<action itemid="6120" script="other/spellbook.lua" />
 	<action fromid="8900" toid="8904" script="other/spellbook.lua" />
 	<action itemid="8918" script="other/spellbook.lua" />
-
+	<action itemid="12647" script="other/spellbook.lua" />
+	<action itemid="16112" script="other/spellbook.lua" />
+	<action itemid="18401" script="other/spellbook.lua" />
+	<action fromid="22422" toid="22424" script="other/spellbook.lua" />
+
```
Comment (ranisalt) [inline data/actions/actions.xml]: Trailing whitespace

**Comment by ranisalt** [comment]
Do you have a git client installed locally? I don't think it's possible to squash commits via GitHub tools but you seem to be editing the files directly in GitHub editor.

**Comment by l04d** [comment]
@ranisalt actually I'm waiting my interval to reboot and squash via terminal, on windows I only can via github tool, and actually I was trying and i closed to do it after as i noticed i just can't do it through tools.

**Pull request closed without merging**